pybricksdev/connections/pybricks.py

```diff
@@ -227,19 +473,11 @@ def _pybricks_service_handler(self, _: int, data: bytes) -> None:
             if self._enable_line_handler:
                 self._handle_line_data(payload)
 
-    async def connect(self, device: BLEDevice):
-        """Connects to a device that was discovered with :meth:`pybricksdev.ble.find_device`
-
-        Args:
-            device: The device to connect to.
-
-        Raises:
-            BleakError: if connecting failed (or old firmware without Device
-                Information Service)
-            RuntimeError: if Pybricks Protocol version is not supported
-        """
-        logger.info(f"Connecting to {device.name}")
+    def _handle_disconnect(self):
+        logger.info("Disconnected!")
+        self.connection_state_observable.on_next(ConnectionState.DISCONNECTED)
 
+    async def connect(self):
```

Comment:
The log is moved into the transport so that it can properly retrieve the device name.

The docstrings don't seem applicable anymore because the args are different and exceptions can occur in different places (BleakError in the _BLETransport and RuntimeError when analyzing protocol versions).

Reply:
I would rather fix the docstrings if we need to than remove them. Especially if we are changing behavior.